src/base/AsyncJob.cc

```diff
@@ -199,15 +197,16 @@ AsyncJob::ReportAllJobs(StoreEntry *e)
 {
     PackableStream os(*e);
     // this loop uses YAML syntax, but AsyncJob::status() still needs to be adjusted to use YAML
-    const char *indent = "    ";
+    const SBuf indent("  ");
     for (const auto job: AllJobs()) {
-        os << indent << job->id << ":\n";
-        os << indent << indent << "type: '" << job->typeName << "'\n";
-        os << indent << indent << "status:" << job->status() << '\n';
-        if (!job->started_)
-            os << indent << indent << "started: false\n";
+        os << job->id << ":\n";
+        os << indent << "type: '" << job->typeName << "'\n";
+        const char *status = job->status();
+        if (status && strlen(status))
+            os << indent << "status: '" << job->status() << "'\n";
+        os << indent << "started: " << (job->started_ ? "Yes": "No" ) << '\n';
         if (job->stopReason)
-            os << indent << indent << "stopped: '" << job->stopReason << "'\n";
+            os << indent << "stopped: \'" << job->stopReason << "'\n";
```

Comment:
single quote should not need to be escaped within double-quoted string:
```suggestion
            os << indent << "stopped: '" << job->stopReason << "'\n";
```